addons/pharma/stringtable.xml

```diff
@@ -553,21 +525,7 @@
             <Finnish>Paina TXA</Finnish>
         </Key>
         <Key ID="STR_KAT_Pharma_Norep_DescShort">
-            <English>Used to raise blood pressure</English>
-            <German>Wird zur Erhöhung des Blutdrucks verwendet</German>
-            <Polish>Używana do podniesienia ciśnienia krwi</Polish>
-            <French>Utilisé pour remonter la pression artérielle</French>
-            <Chinese>用于提高血压</Chinese>
-            <Chinesesimp>用于提高血压</Chinesesimp>
-            <Italian>Usato per aumentare la pressione sanguigna</Italian>
-            <Spanish>Se utiliza para subir la presión arterial</Spanish>
-            <Korean>혈압을 높이는 데 사용</Korean>
-            <Turkish>Kan basıncını yükseltmek için kullanılır</Turkish>
-            <Czech>Používáno ke zvýšení krevního tlaku</Czech>
-            <Japanese>血圧を上げるために使用される</Japanese>
-            <Russian>Используется для повышения артериального давления</Russian>
-            <Portuguese>Usado para aumentar a pressão arterial</Portuguese>
-            <Finnish>Käytetään verenpaineen nostamiseen</Finnish>
+            <English>Slows bleeding, decreases transfusion speed, raises blood pressure</English>
```

Comment:
maybe we should use stuff like "narrows a blood vessels"? Not sure tbh

Reply:
i feel like saying vasodilator is a bad idea since all people thing these meds do now is raise or lower BP but its not that simple in KAT.